src/DotNetWorker.Grpc/GrpcServiceCollectionExtensions.cs

```diff
@@ -75,6 +78,7 @@ public static IServiceCollection AddGrpc(this IServiceCollection services)
                 .Configure<IConfiguration>((arguments, config) =>
                 {
                     config.Bind(arguments);
+                    arguments.ScriptRoot = config["AzureWebJobsScriptRoot"];
```

Comment:
If we use the same value as the property name (`AzureWebJobsScriptRoot` instead of `ScriptRoot`), we do not need this line. Is that rite?

Reply:
@brettsam and I picked the shorter name when he was explaining to me how to get this value passed to GrpcWorker. I think we kept the shorter name for simplicity, but open to changing it to `AzureWebjobsScriptRoot`. The two names are equally understandable to me, but if anyone thinks the latter is more descriptive/understandable then I'm happy to change it and also clean up this line.